src/components/ClassRoster/columns.jsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import { Hyperlink } from '@edx/paragon';
+import { MfeBaseUrl } from '../../constants';
 
-const columns = (componentNavigationHandler) => {
+const columns = (courseId, setRosterStudent, history) => {
```

Comment:
Uhm I'm not sure about this, and my reason is, what will happen if `setRosterStudent` is not provided? Whell in general what will happen if for some reason I forgot to provide any argument to this function? I would recommend to add a default value, just in case.


This is an example
```suggestion
const columns = ({
courseId = '', 
setRosterStudent = () => {}, 
history = { push: () => {}
}
} = {}) => {
```